python/paddle/fluid/tests/unittests/test_fc_op.py

```diff
@@ -131,6 +133,34 @@ def config(self):
         self.matrix = MatrixGenerate(1, 4, 3, 128, 128, 2)
 
 
+class TestFcAPI(unittest.TestCase):
+    # test for parameter num_flatten_dims=-1
```

Comment:
Rename class "TestFcAPI" to "TestFcOp_NumFlattenDims_NegOne" or  "TestFcOpWithNegOneNumFlattenDims"then delete comment at line 137.

Reason:
1. TestFcAPI doesn't contain lots of information considering that there is a class TestFcOp in the same file
2. After renaming, the class speaks the meaning itself.

Reply:
Thanks. Done